app/components/page/container_component.html.erb

```diff
@@ -1,4 +1,8 @@
 <%= render UI::ContainerComponent.new do %>
+  <% if breadcrumbs? %>
+    <%= render breadcrumbs %>
+  <% end %>
```

Comment:
I feel like this would be a good fallback, that way you could still override it, but you don't have to pass in the `with_breadcrumb` on every page. I feel like we could also do something similar for the title, but that's for another PR.

```suggestion
  <% if breadcrumbs? %>
    <%= render breadcrumbs %>
  <% else %>
    <%= new UI::BreadcrumbsComponent.new(page: @page) %>
  <% end %>
```